docs/source/setup-guide.rst

```diff
@@ -80,21 +91,49 @@ as a useful "minimal" system.
 .. code-block:: bash
 
    # Go to the proj_template directory
-   $ cd CFU-Playground
    $ cd proj/proj_template
 
    # Start from an empty build
    $ make clean
 
    # Program the bitstream onto the board. The first run will take several minutes
    # as Vivado synthesizes a bitstream
+   # If this works fine, you will get a chasing LED pattern with the 4 LEDs
    $ make prog
 
    # Build the RISCV program and load it onto the board
-   # Also starts a terminal
+   # Also starts a terminal ( exit the terminal with CTRL+\ )
```

Comment:
I didn't know about Ctrl+\ - I have been mashing ctrl-c. 

Hmm. I just tried Ctrl+\ and found it left my terminal in a bad state. Maybe we should recommend mashing ctrl-C? WDYT?

Reply:
CTRL+C is better actually, I changed it.

Reply:
The trick is that you need *two* Ctrl-C's to exit litex_term, which may be why Marc originally resorted to Ctrl+\ .    (I'm just catching up on stuff now after returning from vacation.)